torch/_inductor/ir.py

```diff
@@ -3016,9 +3016,27 @@ def create(
         output_padding_: List[int],
         groups: int,
     ):
+        with torch._subclasses.FakeTensorMode():
+            x_fake = ir_node_to_tensor(x, guard_shape=False)
+            weight_fake = ir_node_to_tensor(weight, guard_shape=False)
+            bias_fake = (
+                ir_node_to_tensor(bias, guard_shape=False) if bias is not None else bias
+            )
+            output = torch.ops.aten.convolution(
```

Comment:
If we add this path to query output strider order, do we still need the following path: 

````
  with FakeTensorMode():
            output, *_ = cls.process_kernel(
                torch.ops.aten.convolution,
                x,
                weight,
                bias,
                stride,
                padding,
                dilation,
                transposed,
                output_padding,
                groups,
            )
```